src/actions/general.test.js

```diff
@@ -132,6 +132,24 @@ describe('Actions.General', () => {
         assert.equal(timezones.length === 0, false);
     });
 
+    it('getWarnMetricsStatus', async () => {
+        const responseData = {
+            metric1: true,
+            metric2: false,
+        };
+
+        nock(Client4.getBaseRoute()).
+            get('/warn_metrics/status').
+            query(true).
+            reply(200, responseData);
+
+        await Actions.getWarnMetricsStatus()(store.dispatch, store.getState);
+        await TestHelper.wait(100);
```

Comment:
I'm not sure this `wait` is needed here, for sure shouldn't be needed, but I'm not sure if we need more changes anywhere to make it work properly.

Reply:
I'll remove it, but right now it looks to be used in some places and not in others, so I erred on the caution side.